core/src/kura.rs

```diff
@@ -720,6 +806,38 @@ mod tests {
 
     use super::*;
 
+    fn indices<const N: usize>(value: [(u64, u64); N]) -> [BlockIndices; N] {
+        let mut ret = [BlockIndices {
+            start: 0,
+            length: 0,
+        }; N];
+        for idx in 0..value.len() {
+            ret[idx] = value[idx].into();
+        }
+        ret
+    }
+
+    impl PartialEq for BlockIndices {
```

Comment:
Why not derive it?

Reply:
Because it's under `cfg(test)` we don't actually need this outside of syntactic sugar.